nucypher/policy/policies.py

```diff
@@ -199,54 +153,36 @@ def worker(address):
         try:
             successes = worker_pool.block_until_target_successes()
         except (WorkerPool.OutOfValues, WorkerPool.TimedOut):
-            # It's possible to raise some other exceptions here,
-            # but we will use the logic below.
+            # It's possible to raise some other exceptions here but we will use the logic below.
             successes = worker_pool.get_successes()
         finally:
             worker_pool.cancel()
             worker_pool.join()
-
-        accepted_arrangements = {ursula: arrangement for ursula, arrangement in successes.values()}
         failures = worker_pool.get_failures()
 
-        accepted_addresses = ", ".join(ursula.checksum_address for ursula in accepted_arrangements)
-
-        if len(accepted_arrangements) < self.shares:
-
-            rejected_proposals = "\n".join(f"{address}: {value}" for address, (type_, value, traceback) in failures.items())
-
-            self.log.debug(
-                "Could not find enough Ursulas to accept proposals.\n"
-                f"Accepted: {accepted_addresses}\n"
-                f"Rejected:\n{rejected_proposals}")
-
-            raise self._not_enough_ursulas_exception()
-        else:
-            self.log.debug(f"Finished proposing arrangements; accepted: {accepted_addresses}")
-
-        return accepted_arrangements
-
-    def enact(self,
-              network_middleware: RestMiddleware,
-              handpicked_ursulas: Optional[Iterable['Ursula']] = None,
-              ) -> 'EnactedPolicy':
-        """
-        Attempts to enact the policy, returns an `EnactedPolicy` object on success.
-        """
+        accepted_addresses = ", ".join(ursula.checksum_address for ursula in successes.values())
+        if len(successes) < self.shares:
+            rejections = "\n".join(f"{address}: {value}" for address, (type_, value, traceback) in failures.items())
+            message = "Could not find enough Ursulas to accept proposals.\n"\
```

Comment:
May be better phrased as something like "Could not find enough online Ursulas", since we didn't really propose anything.